specification/securityandcompliance/resource-manager/Microsoft.SecurityAndCompliance/stable/2021-01-11/privateLinkServicesForM365ComplianceCenter.json

```diff
@@ -0,0 +1,1500 @@
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "name": "privateLinkServicesForM365ComplianceCenterDescription",
+            "in": "body",
+            "description": "The service instance metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/privateLinkServicesForM365ComplianceCenterDescription"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Put request accepted and an existing resource is being updated; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/privateLinkServicesForM365ComplianceCenterDescription"
+            }
+          },
+          "201": {
+            "description": "Created - Put request accepted and a new resource was created; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/privateLinkServicesForM365ComplianceCenterDescription"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "Resource"
+        ],
+        "description": "Update the metadata of a privateLinkServicesForM365ComplianceCenter instance.",
+        "operationId": "privateLinkServicesForM365ComplianceCenter_Update",
+        "x-ms-examples": {
+          "Patch service": {
+            "$ref": "./examples/ComplianceCenterServicePatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "name": "servicePatchDescription",
+            "in": "body",
+            "description": "The service instance metadata and security metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServicesPatchDescription"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The tags for the resource were updated successfully.",
```

Comment:
>The tags [](start = 28, length = 8)

According to your definition you are updating this definition as well:

ServicesPropertiesUpdateParameters

If yes, remove "The tags" from the description as it is a little misleading (review this for all your PATCH operations)

Reply:
done